website/src/theme/CodeBlock/index.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
for these copied code, do we wanna put our copy right instead? Then add a blob mentioning where it was copied from both for documentation and copyright attribution back to Facebook 👍

Reply:
@hlminh2000 for sure! Do you know if there is a specific format I need to use for attribution?

Reply:
maybe something like `This code was originally copied from Facebook at ___ and modified under MIT license`?